component-dev-best-practice.md

```diff
@@ -168,25 +168,25 @@ You can find an [IA-DoD template here](#integration-application-definition-of-do
 * All development branches are reviewed and merged into master branch
 * Sprint review preparation is finished.
 
-####Integration application Quality Assurance
+#### Integration application Quality Assurance
 * Test cases for the Integration application are ready at the TestRail
 * Functional testing is finished
 * Regression testing is finished (optional)
 * Integration application is deployed to the required stage
   * Smoke test is finished
 * Release notes are done.
 
-##Code Library development
-###Code Library development prerequisites
+## Code Library development
+### Code Library development prerequisites
 * PS team has access to the source with all requirements
 * Meeting with Platform team representatives regarding Code Library re-usability
 * Access to the needed services/code with all needed permissions is granted
 * No open tasks that could block Code Library development process
 * Code Library architecture should be discussed and approved before start coding.
 
-###Definition of Done (CL-DoD)
+### Definition of Done (CL-DoD)
```

Comment:
maybe `Code Library Definition of Done (CL-DoD)` instead of `Definition of Done (CL-DoD)`